### PR TITLE
kubectl: update to 1.16.2 and 1.15.4

### PR DESCRIPTION
**Pull request by szczad**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G95
Xcode 11.0 11A420a

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/kubectl/Portfile [hunk 1]
```diff
@@ -62,7 +69,7 @@ subport kubectl-1.10 {
 if {${subport} == ${name}} {
     PortGroup           obsolete 1.0
     replaced_by         ${latestVersion}
-    version             1.15.0
+    version             1.16.1
     revision            1
```
Comment (mf2k) [inline sysutils/kubectl/Portfile]: The `revision` line needs to be reset to 0 when upgrading the `version`.

Comment (mf2k) [reply sysutils/kubectl/Portfile]: Given that this sub-port is labelled obsolete and `replaced_by` the latest version, should it even be updated at all?

**Comment by neverpanic** [comment]
@szczad ping? Could you fix the review comments?

**Comment by pmetzger** [comment]
@szczad Unfortunately, to make progress on this, you will need to update your pull request. If we don't hear from you soon, we may have to close the pull request, at least until you re-appear.

**Comment by szczad** [comment]
I'm deeply sorry for inconvenience. Unfortunately due to tremendous amount of side work I couldn't focus on this particular task.

**Comment by szczad** [comment]
@mf2k I'll be honest - I have no idea why original author made it this way. But according to the advice I've made relevant change.

**Comment by lbschenkel** [comment]
Hi, thank you for your implementing the changes.

A final nitpick, if you don't mind: can you just rewrite the commit message to say `kubectl` (note the first letter lowercase) to match the port name exactly? This will match the standard for commit messages in this project.

@mf2k: your comments were addressed. If I don't hear any NAK in the next 24h, I'm going to merge this.

**Comment by szczad** [comment]
@lbschenkel If that is the standard then of course.

File: sysutils/kubectl/Portfile [hunk 2]
```diff
@@ -62,8 +70,8 @@ subport kubectl-1.10 {
 if {${subport} == ${name}} {
     PortGroup           obsolete 1.0
     replaced_by         ${latestVersion}
-    version             1.15.0
-    revision            1
+    version             1.16.0
+    revision            2
```
Comment (mf2k) [inline sysutils/kubectl/Portfile]: The `revision` line needs to be reset to 0 when upgrading the `version`.

**Pull request merged**